README.md

```diff
@@ -244,7 +244,7 @@ Make sure that you have the following required dependencies installed:
 
 * CMake v3.5.1 or higher
 
-* [Go](https://golang.org/) 1.14.x or higher
+* [Go](https://golang.org/) 1.14.1 or higher
```

Comment:
is there a specific 1.4.1 feature, or that it matches fabric has published in 2.1.0 ?

Reply:
I just used 1.14.1 as this is what fabric explicitly called out,